samples/extension-module/dummy/services/DummyService.ts

```diff
@@ -1,3 +1,4 @@
+import type { Query } from '../../../../../aries-framework-javascript/packages/core/src/storage/StorageService'
```

Comment:
This import should not depend on the directory the repo is cloned. Ideally, it should be imported directly from @aries-framework/core:

```suggestion
import type { Query } from '@aries-framework-javascript/core'
```

For that purpose, you will need to expose `Query` in **packages/core/src/index.ts**: change line 20 to `export { StorageService, Query } from './storage/StorageService'`

Reply:
On it